tests/integration/kubernetes/runtimeclass_workloads/tests-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kata-containers-k8s-tests
```

Comment:
If I am not asking too much, the namespace name could be random :)

Reply:
Why? :-)

I mean, it makes things more complicated without any clear benefit.

Let's say, we create a random namespace ... as we don't run any kind of tests in parallel (as in more than one execution of the same job in the same machine at the same time), why having a random namespace would be better?

Reply:
If for some unexpected termination of the job the namespace didn't get remove, which random namespace we are still protected. But I might being too paranoiac here :)